modules/prelude-dart.el

```diff
@@ -0,0 +1,59 @@
+;;; prelude-dart.el --- Emacs Prelude: Clojure programming configuration.
+;;
+;; Copyright © 2011-2020 Bozhidar Batsov
+;;
+;; Author: Rafael Medina <rafaelmedina789@gmail.com>
+;; URL: http://batsov.com/prelude
+;; Version: 1.0.0
+;; Keywords: convenience
+
+;; This file is not part of GNU Emacs.
+
+;;; Commentary:
+
+;; Some basic configuration for prelude-dart.
+
+;;; License:
+
+;; This program is free software; you can redistribute it and/or
+;; modify it under the terms of the GNU General Public License
+;; as published by the Free Software Foundation; either version 3
+;; of the License, or (at your option) any later version.
+;;
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with GNU Emacs; see the file COPYING.  If not, write to the
+;; Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+;; Boston, MA 02110-1301, USA.
+
+;;; Code:
+
+(prelude-require-packages '(lsp-mode lsp-dart lsp-ui))
```

Comment:
Don't those LSP deps come from `prelude-lsp.el` already? I guess you just need to require it here.